src/main/java/net/minecraftforge/fluids/BlockFluidClassic.java

```diff
@@ -92,53 +102,46 @@ public int getMaxRenderHeightMeta()
         return 0;
     }
 
-    @Override
-    public int getLightValue(@Nonnull IBlockState state, @Nonnull IBlockAccess world, @Nonnull BlockPos pos)
```

Comment:
What was the purpose of `getLightValue` before, and why remove it?

Reply:
It's the override that's removed here, `BlockFluidBase` now has a more general implementation that covers this as well as `BlockFluidFinite`.